src/swaggers/openpix.yml

```diff
@@ -1207,6 +1214,17 @@ paths:
                     comment: payment comment
                     correlationID: payment1
                     sourceAccountId: my-source-account-id
+                  transaction:
+                    value: 100
+                    endToEndId: transaction-end-to-end-id
+                    time: 2023-03-20T13:14:17.000Z
```

Comment:
Is this correct? shouldn't it be a string?

Reply:
it should, idk why generated like that

Reply:
yml just works that way, on the docs is right

Reply:
![Uploading Screenshot 2023-08-07 at 14.30.02.png…]()

Reply:
I would rather keep a pattern, everything string or not string, I don't like the mixing tbh